### PR TITLE
List available games on games homepage

### PR DESCRIPTION
**Pull request by scottAF**

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 class GamesController < ApplicationController
 
   def index
+    @games = Game.all
```
Comment (talk2MeGooseman) [inline app/controllers/games_controller.rb]: Oops just noticed your not using available scope here to filter games here

File: app/views/games/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h1>Available Games</h1>
+
+<h2> 
+  <% @games.each do |game| %>
+    <% if game.white != nil && game.black == nil %>
```
Comment (talk2MeGooseman) [inline app/views/games/index.html.erb]: This shouldn't be necessary if you use the scope

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'games#show'
+  root 'users#index'
+    resources :games, only: [:show, :index]
```
Comment (talk2MeGooseman) [inline config/routes.rb]: Indent issue here

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (scottAF) [reply app/controllers/games_controller.rb]: https://github.com/scottAF/ChestMasters/pull/15/commits/0d5d841fd174ba17e1ba210d0a2d5cad89a293d3

File: app/views/games/index.html.erb (hunk 2 above)
Comment (scottAF) [reply app/views/games/index.html.erb]: https://github.com/scottAF/ChestMasters/pull/15/commits/0d5d841fd174ba17e1ba210d0a2d5cad89a293d3

File: config/routes.rb (hunk 3 above)
Comment (scottAF) [reply config/routes.rb]: https://github.com/scottAF/ChestMasters/pull/15/commits/14550d12615c435443fedcaad525aaf57a006dd7

**Pull request merged**